packages/compass-settings/src/components/settings/theme.tsx

```diff
@@ -29,13 +29,27 @@ const checkboxStyles = css({
   marginBottom: spacing[3],
 });
 
+const radioBoxStyles = css({
+  div: {
+    textAlign: 'left',
+    padding: spacing[3],
+    justifyContent: 'flex-start',
+  },
+});
+
+const themePreviewStyles = css({
+  marginRight: spacing[2],
+  maxWidth: '50%',
```

Comment:
Kinda arbitrary, but should leave enough space for the text to the right of it in both cases and at all the sizes we support.